std/typecons.d

```diff
@@ -5141,7 +5141,16 @@ therefore avoiding the overhead of $(D new). This facility is unsafe;
 it is the responsibility of the user to not escape a reference to the
 object outside the scope.
 
-Note: it's illegal to move a class reference even if you are sure there
+The class destructor will be called when the result of `scoped()` is 
+itself destroyed.
+
+Scoped class instances can be embedded in a parent `class` or `struct`, 
+just like a child struct instance. Scoped member variables must have 
+type `typeof(scoped!Class(args))`, and be initialized with a call to 
+scoped. See below for an example.
+
+Note:
+It's illegal to move a class reference even if you are sure there
```

Comment:
I still don't quite understand exactly what that means, though. How do you "move an object's data" if said object is being held by `Scoped`? Copy it elsewhere and invalidate the scoped reference? Manually overwrite the scoped reference pointer? I'm not seeing the point of that comment. Obviously, I'm missing something obvious, but that just shows how badly written that comment is? :-P